src/Aspire.Dashboard/Components/ResourcesGridColumns/SourceColumnDisplay.razor

```diff
@@ -13,7 +13,7 @@
                PreCopyToolTip="@Loc[nameof(Columns.SourceColumnSourceCopyFullPathToClipboard)]"
                ToolTip="@projectPath" />
 }
-else if (Resource.TryGetExecutablePath(out var executablePath) && Resource.TryGetWorkingDirectory(out var workingDirectory))
```

Comment:
Can we now remove the `TryGetWorkingDirectory` method? I doubt it's used anywhere else.

Reply:
https://github.com/dotnet/aspire/pull/1856